### PR TITLE
Change to font-awesome icons (except js)

### PR DESCRIPTION
**Pull request by soulgalore**
I've tested to changed the icons to use Font Awsome (except the JS version that comes from https://www.iconfinder.com/icons/282802/javascript_js_icon). Hopefully next Font Awesome will contain even better icons we can use. There's an HTML5 and CSS3 logo in there today but will people recognize them?

Here's an example of what it looks like now:
screen shot 2016-12-19 at 16 42 59

It isn't a super big change, let see what you think @micmro and @tobli #63

### CODE REVIEW

**Comment by tobli** [comment]
I think they look really crisp! Some initial feedback:
- it'd be super if you could fix the formatting so `npm run lint` passes. I'll setup Travis later so it'll check it for you.
- I think the "other" mime icon could be inverted. The rest of the mime icons are black symbol on white background (so maybe black "?" and a thin black circle outline?). It'll make it easier to separate from the error icons.
- I noticed some errors have the same icon now (e.g. noGzip and noCache). I don't have a strong opinion on this, but I'd like to hear from @micmro.

**Comment by soulgalore** [comment]
yep it will be the same for 400 and 500 errors so we will loose some functionality :(

**Comment by soulgalore** [comment]
Updated the question marks to a standalone without circle, the with a circle is missing from the repo, hopefully it will be there when 5.0 is released.

screen shot 2016-12-19 at 20 44 24

File: src/ts/helpers/icons.ts [hunk 1]
```diff
@@ -14,101 +14,197 @@ const toSvg = (x: number, y: number, title: string, className: string, scale: nu
...
-}
+  return toSvg(x, y, title, "icon-flash", scale, `  <path d="m 13.724296,4.737962 q 0.194716,0.216309 0.07572,0.475924 L 7.958654,17.729559
+  Q 7.818031,18 7.504329,18 7.461078,18 7.352885,17.97846 7.16899,17.924378 7.0770425,17.772918 6.9850949,17.621512 7.0283513,17.4484 L
+  9.15937,8.708015 4.7675305,9.800549 q -0.043251,0.01077 -0.1298072,0.01077 -0.1947161,0 -0.3353388,-0.118981 -0.1947107,-0.162286
+  -0.140628,-0.4219 L 6.3360428,0.34617 Q 6.3792939,0.194711 6.5091226,0.097382 6.6389298,0 6.8120043,0 l 3.5480877,0 q 0.205532,0
+  0.346154,0.135193 0.140628,0.135248 0.140628,0.319132 0,0.08656 -0.05409,0.194711 l -1.849763,5.008456 4.283664,-1.06011 q
+  0.08654,-0.02154 0.129807,-0.02154 0.205532,0 0.367791,0.162285 z" />`) }
```
Comment (tobli) [inline src/ts/helpers/icons.ts:208]: Just missing a newline at the end to pass the lint check.

**Review by micmro — APPROVED** [review]
Looks all good. Just one tiny thing, is there a reason for the for the white space in front of the `<path>`  node: `\`  <path`?

**Comment by micmro** [comment]
Cool - great job! Quite like the icons. What do you think of using a dark grey (perhaps `#666`) for the icons and some dark hue of red (e.g. `#b55`) for the warning one, just to have some more distinction?

**Comment by soulgalore** [comment]
Ooops fixed those spaces. Here is what is look like with grey icons and red warnings:
screen shot 2016-12-20 at 05 52 07

**Comment by micmro** [comment]
Thanks a lot! What do you think about the colours? I think this could work.

**Comment by soulgalore** [comment]
Yes it is good!

**Comment by tobli** [comment]
I think this is a clear improvement. Let's merge this and continue to tweak later. Thanks!

**Pull request merged**